.circleci/config.yml

```diff
@@ -1,11 +1,7 @@
-version: 2
-
+version: 2.1
 defaults: &defaults
   docker:
-    - image: circleci/golang:1.12.4-node
-      auth:
-        username: $DOCKERHUB_USERNAME
-        password: $DOCKERHUB_TOKEN
+    - image: cimg/go:1.17.1-node
```

Comment:
https://circleci.com/developer/images/image/cimg/go - it's the new replacement for the legacy one we were using